src/server/coprocessor/aggregate.rs

```diff
@@ -59,3 +61,36 @@ impl AggrFunc for First {
         Ok(())
     }
 }
+
+struct Sum {
+    res: Option<Datum>,
+}
+
+impl AggrFunc for Sum {
+    fn update(&mut self, mut args: Vec<Datum>) -> Result<()> {
+        if args.len() != 1 {
+            return Err(box_err!("sum only support one column, but got {}", args.len()));
+        }
+        let a = args.pop().unwrap();
+        if a == Datum::Null {
+            return Ok(());
+        }
+        let res = match self.res.take() {
+            Some(b) => box_try!(evaluator::eval_arith(a, b, Datum::checked_add)),
```

Comment:
When meet the first not null row, b will be null. Will eval_arith return null for a+b?